tools/public_api_guard/cdk/scrolling.d.ts

```diff
@@ -163,18 +163,25 @@ export declare class FixedSizeVirtualScrollStrategy implements VirtualScrollStra
     updateItemAndBufferSize(itemSize: number, minBufferPx: number, maxBufferPx: number): void;
 }
 
+export interface Scrollable {
+    elementScrolled(): Observable<Event>;
+    getElementRef(): ElementRef<HTMLElement>;
+    measureScrollOffset(from: ScrollOffsetEdge): number;
+    scrollTo(options: ExtendedScrollToOptions): void;
+}
+
 export declare class ScrollDispatcher implements OnDestroy {
     protected _document?: Document;
     _globalSubscription: Subscription | null;
-    scrollContainers: Map<CdkScrollable, Subscription>;
+    scrollContainers: Map<Scrollable, Subscription>;
     constructor(_ngZone: NgZone, _platform: Platform,
     document?: any);
-    ancestorScrolled(elementRef: ElementRef, auditTimeInMs?: number): Observable<CdkScrollable | void>;
-    deregister(scrollable: CdkScrollable): void;
-    getAncestorScrollContainers(elementRef: ElementRef): CdkScrollable[];
+    ancestorScrolled(elementRef: ElementRef, auditTimeInMs?: number): Observable<Scrollable | void>;
+    deregister(scrollable: Scrollable): void;
+    getAncestorScrollContainers(elementRef: ElementRef): Scrollable[];
     ngOnDestroy(): void;
-    register(scrollable: CdkScrollable): void;
-    scrolled(auditTimeInMs?: number): Observable<CdkScrollable | void>;
```

Comment:
@crisbeto I think the error might be coming from this, then. I'm seeing errors coming from code like
```ts
scrollDispatcher.scrolled().pipe(filter((event: CdkScrollable|void) ...
```
saying
```
Type 'void | Scrollable' is not assignable to type 'void | CdkScrollable'.
```

Reply:
I'm guessing it's because the public properties of the interface aren't exactly the same as `CdkScrollable`. E.g. the interface doesn't have `ngOnInit` and `ngOnDestroy`. How many targets did this break? The correct solution would be to update their app so it references the interface, but we might have to re-evaluate if it's too many.

Reply:
About 500 newly broken with thousands of potential new failures :(

Reply:
Hmm, but they might be targets referencing the same piece of code. Not sure how many unique lines of code are breaking. I'll investigate and get back to you

Reply:
No yeah, the scrollable api is used in hundreds of different places.

I'm also seeing errors where the value returned from `scrollDispatcher.getAncestorScrollContainers` (which is now type `Scrollable`) is being passed as an arg to another function which is expecting type `CdkScrollable`.

Reply:
Alright, let me see if I can make this less break-ey.